modules/raft/src/main/java/org/apache/ignite/raft/jraft/entity/codec/v1/V1Decoder.java

```diff
@@ -51,98 +52,127 @@ public LogEntry decode(final byte[] content) {
         return log;
     }
 
+    // Refactored to look closer to Ignites code style.
     public void decode(final LogEntry log, final byte[] content) {
-        // 1-5 type
-        final int iType = Bits.getInt(content, 1);
-        log.setType(EnumOutter.EntryType.forNumber(iType));
-        // 5-13 index
-        // 13-21 term
-        final long index = Bits.getLong(content, 5);
-        final long term = Bits.getLong(content, 13);
+        var reader = new Reader(content);
```

Comment:
Probably some of the comments should be retained. Or maybe we can describe the format of the entry here (like we do in storages)

Reply:
Ok, I'll explain data format in documentation of the _factory_